addons/mass_mailing/static/src/xml/mass_mailing.xml

```diff
@@ -8,10 +8,10 @@
         </t>
     </div>
     <div t-name="mass_mailing.theme_selector_new" class="o_mail_theme_selector_new">
-        <div t-attf-class="o_mailing_template_message text-white text-start mx-4 pt-3 px-1 #{templates.length ? 'd-none': ''}">
+        <div t-attf-class="o_mailing_template_message text-white text-start mx-4 pt-2 px-1 #{templates.length ? 'd-none': ''}">
             Click on the ⭐ next to the subject to save this mailing as a <span t-esc="modelName"/> template
         </div>
-        <div class="d-inline-block w-100">
+        <div class="d-inline-block w-100" t-if="template">
```

Comment:
Shouldn't it be templateS here? Or maybe I missed something.

Reply:
Ouch, that's embarassing. Yeah, I'll check that it works WITH templates too before I ping you again..

Thanks for the review 😅